common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -53,6 +53,16 @@ public abstract class AzureActiveDirectoryAudience {
     public static final String ALL = "common";
     public static final String MSA_MEGA_TENANT_ID = "9188040d-6c67-4c5b-b112-36a304b66dad";
 
+    // The values here are audience values reported in telemetry
+    // rather than the values used to send to eSTS.
+    public enum AudienceTelemetryConstant {
+        MSA,
+        COMMON,
+        AAD,
+        UNKNOWN;
+    }
```

Comment:
I think we should move this and the method to broker itself. Telemetry is for broker only. 

At this layer it's extra stuff present in AzureActiveDirectoryAudience

Reply:
+1